### PR TITLE
Correct minor typos on landing page

### PR DESCRIPTION
**Pull request by rshah713**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -154,11 +154,11 @@ <h2>CircuitPython is based on Python</h2>
            srcset="{{ "/assets/images/raspberry_pi.png" | relative_url }} 1x,
                    {{ "/assets/images/raspberry_pi@2x.png" | relative_url }} 2x,
                    {{ "/assets/images/raspberry_pi@3x.png" | relative_url }} 3x"  alt="Raspberry Pi">
-      <h2>CircuitPython libraries on single board computers</h2>
+      <h2>CircuitPython libraries on single-board computers</h2>
```
Comment (makermelissa) [inline index.html:157]: Either spelling is correct, but for the sake of consistency with other documentation, I think this should remain without the hyphen.

File: index.html [hunk 2]
```diff
@@ -154,11 +154,11 @@ <h2>CircuitPython is based on Python</h2>
...
       <p>
-        Most CircuitPython libraries also work on <a href="/blinka">single board computers (SBCs)</a>, such as the Raspberry Pi.
+        Most CircuitPython libraries also work on <a href="/blinka">single-board computers (SBCs)</a>, such as the Raspberry Pi.
```
Comment (makermelissa) [inline index.html:159]: Same here.

File: index.html [hunk 3]
```diff
@@ -172,7 +172,7 @@ <h2>Just the beginning...</h2>
     <p>
       CircuitPython continues to evolve, and is constantly being updated. We
       welcome and encourage feedback from the community, and we incorporate this
-      into how we are developing CircuitPython. That's the core of the open source
+      into how we are developing CircuitPython. That's the core of the open-source
```
Comment (makermelissa) [inline index.html:175]: Same thing with open source.

**Review by makermelissa — CHANGES_REQUESTED** [review]
I agree with the changes to low-level and built-in, but would prefer to leave the other changes as they were. Thanks.

**Comment by rshah713** [comment]
Done! Thanks for the review @makermelissa

**Review by makermelissa — APPROVED** [review]
Thanks

**Pull request merged**